Upstream: yes

From 5c5a0f3df9261340725a65f4655a2ab50fd3db4e Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Fri, 8 Apr 2016 09:59:33 -0300
Subject: iproute2: tc_bpf.c: fix building with musl libc
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

We need limits.h for PATH_MAX, fixes:

tc_bpf.c: In function ‘bpf_map_selfcheck_pinned’:
tc_bpf.c:222:12: error: ‘PATH_MAX’ undeclared (first use in this
function)
  char file[PATH_MAX], buff[4096];

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
Acked-by: Daniel Borkmann <daniel@iogearbox.net>
---
 tc/tc_bpf.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tc/tc_bpf.c b/tc/tc_bpf.c
index fe927ac..86c6069 100644
--- a/tc/tc_bpf.c
+++ b/tc/tc_bpf.c
@@ -20,6 +20,7 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <stdarg.h>
+#include <limits.h>
 
 #ifdef HAVE_ELF
 #include <libelf.h>
-- 
cgit v0.12

